piper/buttondialog.py

```diff
@@ -226,6 +240,9 @@ def _on_row_activated(self, listbox, row):
                 print("Unable to grab keyboard, can't set keystroke", file=sys.stderr)
             else:
                 self.stack.set_visible_child_name("capture")
+        elif row is self.row_left_hand_mode:
+            self._action_type = RatbagdButton.ACTION_TYPE_BUTTON
+            self._mapping = -2
```

Comment:
you should make it a const value nonetheless (like a C #define), that way you only have one thing to change in the future. and it's less magic than a -2

Generally I recommend that when you're re-using something that's otherwise a fixed range, choose something well outside that range, e.g. -2000. still obviously visible as non-garbage but much easier to grep for than -2 and less likely to get used by the original value. that's also why the ratbag error codes start at 1000 (errno overlap)